### DISCUSSION TITLE
Front end EC2 instance not connecting

### DISCUSSION THREAD

**Discussion by DOGourav** [learndevopsonline/learndevopsonline#875] [Q&A] [answered]
I. had initially created a t2 micro for front end 
and then as suggested I created a new FE t3 micro for better performance.
But the new server is not connecting from browser.
I check security groups are also attached.
Nginx is installed.

**Comment by pcs1999** [comment]
check if nginx is active or not ?

**Reply by DOGourav** [reply]
it is active
I even tried with a new instance but it is same.

**Reply by pcs1999** [reply]
give ip

**Reply by DOGourav** [reply]
https://18.233.7.235

**Reply by pcs1999** [reply, marked as answer]
http://18.233.7.235/ use this url  im able to connect

https--> will not work 

or directly just paste ip in browser
